Bio/PDB/Residue.py

```diff
@@ -108,6 +108,10 @@ def sort(self):
         by name, with any alternative location specifier for disordered
         atoms (altloc) as a tie-breaker.
         """
+        warnings.warn("The custom sort() method will be deprecated in the "
```

Comment:
change "deprecated" to "removed" (or change the tense to "... has been deprecated in favour of...")